addon/components/sequential-render.js

```diff
@@ -242,11 +255,24 @@ export default Component.extend({
 
   fetchData: task(function* () {
     let { queryParams, taskOptions } = getProperties(this, 'queryParams', 'taskOptions');
-    let content = yield get(this, 'fetchDataTask').perform(queryParams, taskOptions);
```

Comment:
Can we also deprecate asyncRender? and use absense of fetchData as the equivalent of asyncRender=false?

Reply:
Agreed!, marked asyncRender as deprecated in latest commit